*** C:/Users/akasaka/AppData/Local/Temp/TortoiseGit/HttpLineReader-5c3cc999.000.h	Sat Mar  1 10:49:08 2025
--- D:/Code/cd-changer/firm/.pio/libdeps/esp32dev/ArduinoAudioTools/src/AudioTools/CoreAudio/AudioHttp/HttpLineReader.h	Sat Mar  1 01:06:01 2025
***************
*** 67,76 ****
--- 67,77 ----
        }
        if (!is_buffer_owerflow) {
          str[j] = c;
        }
        j++;
+       if(j % 100 == 0) delay(1);
      }
      str[result - 1] = 0;
      if (is_buffer_owerflow) {
        LOGE("Line cut off: %s", str);
      }
